app/models.py

```diff
@@ -96,6 +96,7 @@ def tier(self):
 class Tunnel(db.Model):  # type: ignore
     id = db.Column(db.Integer, primary_key=True)
     port = db.Column(types.ARRAY(types.String()))
+    tcp_ports = db.Column(types.ARRAY(types.Integer()))
```

Comment:
I'm tempted to change this to "allocated_tcp_ports" or something since this is the _ports they've been allocated_ and not the ports they requested.